Packs/OktaITAdmin/Integrations/Okta_IT_Admin/Okta_IT_Admin.py

```diff
@@ -0,0 +1,789 @@
+        "mobilePhone": "phoneNumbers(val.type && val.type=='mobile').[0].value",
+        "employeeNumber": scim_extension + ".employeeNumber",
+        "groups": "groups(val.display).display",
+        "id": "id",
+        "externalId": "externalId",
+        "locale": "locale",
+        "manager": scim_extension + ".manager.value",
+        "lastName": "name.familyName",
+        "nameFormatted": "name.formatted",
+        "firstName": "name.givenName",
+        "nameHonorificPrefix": "name.honorificPrefix",
+        "nameHonorificSuffix": "name.honorificSuffix",
+        "nameMiddleName": "name.middleName",
+        "nickName": "nickName",
+        "organization": scim_extension + ".organization",
+        "password": "password",
+        "photo": "photos(val.type && val.type=='photo').[0].value",
+        "preferredLanguage": "preferredLanguage",
+        "profileUrl": "profileUrl",
+        "thumbnnail": "photos(val.type && val.type=='thumbnail').[0].value",
+        "timezone": "timezone",
+        "title": "title",
+        "userName": "userName",
+        "login": "userName",
+        "userType": "userType"
+    }
+    parsed_scim = dict()
+    for k, v in mapping.items():
+        try:
+            parsed_scim[k] = demisto.dt(scim, v)
+        except Exception:
+            parsed_scim[k] = None
+    return parsed_scim
+
+
+def get_time_elapsed(fetch_time, last_run):
+    now = datetime.now()
+    demisto.info("Okta Last Run: " + str(last_run))
+    if 'time' in last_run:
+        # Get Last run and parse to date format. Workday report will be pulled from last run time to current time
+        last_run_time = last_run['time']
+        # Convert to date format
+        last_run = datetime.strptime(last_run_time, DATE_FORMAT)
+        time_elapsed_in_minutes = (now - last_run).total_seconds() / 60
+    else:
+        # If last run time is not set, data will be pulled using fetch_time
+        # i.e. last 10min if fetch events is set to 10min
+        last_run_time = (now - timedelta(minutes=int(fetch_time))).strftime(
+            DATE_FORMAT)
+        time_elapsed_in_minutes = fetch_time
+
+    return time_elapsed_in_minutes, last_run_time
+
+
+def send_email(subject, body=''):
+    params = demisto.params()
+    smtp_server_host = params.get('smtp_server')
+    smtp_server_port = params.get('smtp_port')
+    from_email = params.get('from_email')
+    to_email = params.get('email_notification_ids')
+
+    # Send email if smtp details are configured
+    if smtp_server_host and smtp_server_port and from_email and to_email:
+        SERVER = SMTP(smtp_server_host, int(smtp_server_port), local_hostname=smtp_server_host)
+        SERVER.ehlo()
+        msg = MIMEText(body)
+        msg['Subject'] = subject
+        msg['From'] = from_email
+        msg['To'] = to_email
+        SERVER.sendmail(from_email, to_email.split(','), msg.as_string())
+        SERVER.quit()
+
+
+'''COMMAND FUNCTIONS'''
+
+
+def test_module(client, args):
```

Comment:
please add here a test that the fetch_time parameter is an integer and fail the test if not (unless it's not configured)

Reply:
Donr